test/sql/updates/post.catalog.sql

```diff
@@ -10,10 +10,12 @@
 \d+ _timescaledb_catalog.chunk_index
 \d+ _timescaledb_catalog.tablespace
 
-\z _timescaledb_cache.*
-\z _timescaledb_catalog.*
-\z _timescaledb_config.*
-\z _timescaledb_internal.*
+SELECT nspname AS Schema,
```

Comment:
This should check initprivs as well? Or is that being done some other place?

Reply:
Good point. No, this is probably the best place. Added a check.